src/components/Section/Section.test.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+
+import { renderWithIntl } from '@folio/stripes-data-transfer-components/test/jest/helpers';
+import '../../../test/jest/__mock__';
+import { fireEvent } from '@testing-library/dom';
+import { translationsProperties } from '../../../test/jest/helpers';
+
+import { Section } from './Section';
+
+const childElement = (
+  <div>
+    <span>child component</span>
+  </div>
+);
+const sectionWithLabel = {
+  label: 'test label',
+  optional: false,
+};
+const sectionWithLabelAndDataTestAttribute = {
+  label: 'test label',
+  rest: 'data-test-tester',
+};
+const sectionWithChildElements = {
+  children: childElement,
+  optional: true,
+};
+const sectionWithoutHeadline = {
+  children: childElement,
+  optional: false,
+};
+const sectionWithCheckboxAndLabel = {
+  label: 'test label',
+  optional: true,
+  isOpen: false,
+  children: [childElement],
+};
+
+const renderSectionContainer = ({
+  label, optional, children, isOpen, rest,
+}) => {
+  const component = (
+    <Section
+      label={label}
+      optional={optional}
+      isOpen={isOpen}
+      {...rest}
+    >
+      {children}
+    </Section>
+  );
+
+  return renderWithIntl(component, translationsProperties);
+};
+
+describe('Section', () => {
+  it('should be rendered only with label', () => {
+    const { getByText } = renderSectionContainer(sectionWithLabel);
+
+    expect(getByText('test label')).toBeDefined();
+  });
+  it('should be rendered without headline', () => {
```

Comment:
Separate test cases with an empty line, please
```suggestion
  });
  
  it('should be rendered without headline', () => {
```